src/components/Thanks/MyKiva/OptInModule.vue

```diff
@@ -0,0 +1,261 @@
+	}
+	if (props.loans.length === 2) {
+		const names = props.loans.map(loan => loan?.name).join(' and ');
+		return `Thank you! You, ${names} are in this together now.`;
+	}
+
+	return `Thank you! You, ${props.loans[0]?.name}, ${props.loans[1]?.name}
+		and ${props.loans[2]?.name} are in this together now.`;
+});
+
+const description = computed(
+	() => `Want to hear how you're impacting ${borrowerName.value}'s life and more ways to help people like them?`
+);
+
+const loansToDisplay = computed(() => {
+	const loans = [...props.loans];
+	if (props.loans.length === 3) {
+		const indexToRemove = loans.indexOf(loan => loan.id === props.selectedLoan.id);
+		const removedLoan = loans.splice(indexToRemove, 1)[0];
+		loans.splice(1, 0, removedLoan);
+	}
+	return loans.slice(0, 3);
+});
+
+const getMarginRight = index => {
+	if (loansToDisplay.value.length > 2 && index === 0) {
+		if (isMobile.value) {
+			return '-40px';
+		}
+		return '-80px';
+	}
+
+	return '0';
+};
+
+const getMarginLeft = index => {
+	if (loansToDisplay.value.length > 1 && index === loansToDisplay.value.length - 1) {
+		if (isMobile.value) {
+			return '-40px';
+		}
+		return '-80px';
+	}
+
+	return '0';
+};
+
+const hash = loan => {
+	return loan?.image?.hash ?? '';
+};
+
+const updateOptIn = value => {
+	if (value) {
+		try {
+			apollo.mutate({
+				mutation: gql`
+					mutation updateCommunicationSettings(
+						$lenderNews: Boolean
+					) {
+						my {
+							updateCommunicationSettings(
+								communicationSettings: {
+									lenderNews: $lenderNews
+								}
+							)
+						}
+					}
+				`,
+				variables: {
+					lenderNews: value,
+				},
+			});
+		} catch (error) {
+			logReadQueryError(error, 'error');
+		}
+	}
+	newConsentAnswered.value = true;
+	receiveNews.value = value;
+};
+</script>
+
+<style lang="postcss" scoped>
+
+.module-container {
+	max-width: 620px;
+	max-height: 800px;
+
+	@apply tw-flex tw-flex-col tw-mx-auto tw-overflow-hidden tw-opacity-full tw-bg-white
+		tw-text-center tw-px-3 md:tw-px-8 tw-gap-3 tw-rounded-lg tw-py-4;
+}
+
+.borrower-container {
+	animation: fadein ease-in 1s;
+	width: 70px;
+
+	@screen md {
+		width: 150px;
+	}
+
+	@apply tw-block tw-relative tw-mx-auto tw-z-4;
+}
+
+.borrower-container > div {
+	height: 100px;
+
+	@screen md {
+		height: 200px;
+	}
+}
+
+.borrower-image {
+	width: 80px !important;
+	height: 80px !important;
+
+	@screen md {
+		width: 160px !important;
+		height: 160px !important;
+	}
+
+	@apply tw-w-full tw-rounded-full tw-bg-black tw-border-4 tw-border-white tw-z-2 !tw-pb-0;
+}
+
+.borrower-image :deep(img) {
+	@apply tw-object-cover;
+}
+
+.centered-borrower {
+	width: 100px !important;
+	height: 100px !important;
+
+	@screen md {
+		width: 180px !important;
+		height: 180px !important;
+	}
+}
+
+.ghost-button :deep(span) {
+	@apply tw-bg-transparent;
+}
+
+.collapse-enter-active,
+.collapse-leave-active {
+  transition: max-height 1s ease, opacity 1s ease;
```

Comment:
When you click the "Yes..." button, you will see the padding immediately disappear from inside the white box, then animate. That makes for a rough animation, so the padding should disappear like it does.

Reply:
This happens with the "No.." link as well